source/Mlos.Python/mlos/Grpc/BayesianOptimizerProxy.py

```diff
@@ -102,17 +108,19 @@ def register(self, feature_values_pandas_frame, target_values_pandas_frame):
         self._optimizer_stub.RegisterObservations(register_request)
 
     @trace()
-    def get_all_observations(self) -> Tuple[pd.DataFrame, pd.DataFrame]:
+    def get_all_observations(self) -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
         response = self._optimizer_stub.GetAllObservations(self.optimizer_handle)
         features_df = pd.read_json(response.Features.FeaturesJsonString, orient='index')
         objectives_df = pd.read_json(response.ObjectiveValues.ObjectiveValuesJsonString, orient='index')
-        return features_df, objectives_df
+        return features_df, objectives_df, None

```

Comment:
```suggestion
        context_df = None
        return features_df, objectives_df, context_df 
```
Also, calling it features_df instead of parameters_df doesn't make sense anymore, but I guess that's too big of a change...

Reply:
Is it?  Would be good to be consistent if we can.

Reply:
Basically I didn't want to change the messages and I left the names here to be consistent with the messages.
Changing the messages is a bigger change.